qcodes/instrument_drivers/oxford/MercuryiPS_VISA.py

```diff
@@ -190,14 +243,73 @@ def __init__(self, name: str, address: str, visalib=None,
         # to ensure a correct snapshot, we must wrap the get function
         self.IDN.get = self.IDN._wrap_get(self._idn_getter)
 
-        # TODO: Query instrument to ensure which PSUs we have
+        # TODO: Query instrument to ensure which PSUs are actually present
```

Comment:
is this an "issue" or "nice-to-have"? in other words, will it be done in this PR? If not, perhaps, make an issue out of it?